<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutSimulation</class>
 <widget class="QDialog" name="AboutSimulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Amphora Simulation</string>
  </property>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>368</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>11</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelAmphoraSimulation">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Amphora Simulation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelVersion">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDeveloper">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Developed by Edward Ruzek</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelBuildDate">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Built on</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>AboutSimulation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
